CLAUDE.md

```
@@ -315,29 +315,241 @@ When using MCP servers (Model Context Protocol):
 
 **Git Workflow:**
 
 - Husky + lint-staged auto-format on commit
 - Pre-commit checks: Prettier format + ESLint on staged TS/JS files
 - Skip hooks only via `git commit -n` (not recommended)
 
+**Branch Naming (Conventional Branch):**
+
+Use [Conventional Branch](https://conventional-branch.github.io/) naming format:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)